examples/consolidated-existing-cloudtrail-in-another-account-datadog-co-existance/security-account.tf

```diff
@@ -0,0 +1,43 @@
+# in a "security" account - ID: 200000000000
+
+provider "lacework" {}
+
+provider "aws" {
+  region = "us-west-2"
+}
+
+# the CT logs live in this account
+# Note: in order to make this co-exist with Datadog (useful for the transitional period)
+#       datadog has to be allowed to maintain the lambda hook on the bucket, so we can't have
+#       LW hook that (it'd be the easiest option)
+resource "aws_s3_bucket" "ct_bucket" {
+  bucket = "org-ct-bucket"
+}
+
+# and so does their KMS Key
+# let's assume its ARN is: arn:aws:kms:us-west-2:200000000000:key/6e2010aa-27e4-49c6-8887-956abc1caeb9
+resource "aws_kms_key" "ct_key" {
+}
+
+module "aws_cloudtrail" {
+  source  = "lacework/cloudtrail/aws"
+  version = "~> 2.8"
```

Comment:
```suggestion
  source  = "../../"
```

Reply:
We refer to the module locally instead of the registry ☝🏽

Reply:
done